helpers/shells/getStorageUsed.sh

```diff
@@ -16,4 +16,4 @@ set -e
 
 # Use jc to output a JSON format with du command
 cd /var/borgwarehouse
-sudo /usr/bin/du -s * | jc --du
\ No newline at end of file
+sudo /usr/bin/du -s ./* | sed 's/\.\///' | jc --du
```

Comment:
What is the reason for this change? Sudo permissions are set strictly. This change cannot be made without modifying the sudo rules (test it, you will have a sudo exception). What's the point ? Thanks to you!

Reply:
Thanks for your explanations. This case cannot occur in a normal context of use of borgwarehouse because the users are necessarily alpha-numeric suites. But I still approve of your change, which is in line with the state of the art and compliance with standards. Thank you.